code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -504,4 +504,37 @@
 	desc = "You'll be the Ruler of the King's Navy in no time."
 	icon_state = "lordadmiral"
 	item_state = "lordadmiral"
-	allowed = list (/obj/item/weapon/gun)
\ No newline at end of file
+	allowed = list (/obj/item/weapon/gun)
+
+
+//Kluys fluff nanofibre suit
+/obj/item/clothing/suit/kluysfluff
+	name = "Nano Fibre Jacket"
+	desc = "A Black Suit made out of nanofibre. The newest of cyberpunk fashion using hightech liquid to solid materials."
+	icon_state = "Kluysfluff1"
+	item_state = "Kluysfluff1"
+	blood_overlay_type = "coat"
+	body_parts_covered = UPPER_TORSO|ARMS
+
+	verb/toggle()

```

Comment:
Line should be `/obj/item/clothing/suit/kluysfluff/verb/toggle()` as relative pathing shouldn't be used unless the entire file is using it.
Additionally, this is in the wrong file. Fluff items should go [here](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/customitems/item_defines.dm).